source/fundamentals/serialization/poco.txt

```diff
@@ -443,11 +480,36 @@ serializing the ``YearBuilt`` property if it is undefined:
    {
        public Guid Id { get; set; }
 
-       [BsonIgnore]
-       public int YearBuilt { get; set; }
+       [BsonIgnoreIfNull]
        public string Style { get; set; }
+       public int YearBuilt { get; set; }
    }
 
+You can also instruct the driver to ignore a property that contains a
+``null`` value when you register the class map, as shown in the
+following example:
+
+.. code-block:: csharp
+   :copyable: true
+   :emphasize-lines: 4
+
+   BsonClassMap.RegisterClassMap<House>(classMap => 
+   {
+      classMap.AutoMap();
+      classMap.MapMember(h => h.Style).SetIgnoreIfNull(true);
+   });
+
+.. note:: Numerical Properties
+
+   You cannot use the ``[BsonIgnoreIfNull]`` attribute or
+   ``SetIgnoreIfNull()`` method to prevent undefined numerical
+   properties from being serialized unless you mark the properties as
```

Comment:
"undefined numerical property" wording might be confusing for .NET users, as numeric (value) values are always defined. 'Uninitialized' or 'default' sound better.